buildah.go

```diff
@@ -23,7 +23,7 @@ const (
 	Package = "buildah"
 	// Version for the Package.  Bump version in contrib/rpm/buildah.spec
 	// too.
-	Version = "1.0"
+	Version = "1.2-dev"
```

Comment:
should this be 1.1 or does this get bumped up by one due to it being seen in master?

Reply:
There are two Patches here.  First one bumps to 1.1, second one to 1.2-dev.  We will tag and release the first one, the second one will just tell us if someone is running buildah off of master

When we release 1.2 we will do the same thing again.  This is how podman releases work.

Reply:
Thanks for the 411.